src/js/control-bar/progress-control/seek-bar.js

```diff
@@ -428,12 +427,5 @@ if (!IS_IOS && !IS_ANDROID) {
   SeekBar.prototype.options_.children.splice(1, 0, 'mouseTimeDisplay');
 }
 
-/**
- * Call the update event for this Slider when this event happens on the player.
- *
- * @type {string}
- */
-SeekBar.prototype.playerEvent = 'timeupdate';
```

Comment:
Thoughts about keeping this in and removing line 84 (`    this.on(this.player_, ['timeupdate', 'ended'], this.update);
)`? update already gets called from ended via the handleEnded and the slider makes sure that the playerEvent handler is added/removed properly.

Reply:
This causes the `update` function to be bound to `timeupdate` before it is throttled in `setEventHandlers_()`, which I don't think we want.

Reply:
Ah, I see.